<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Matrix33Setter</class>
 <widget class="QWidget" name="Matrix33Setter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>72</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-99.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-99.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QDoubleSpinBox" name="doubleSpinBox">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-99.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-99.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_5">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-99.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_6">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-99.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_7">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-99.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_8">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-99.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_9">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-99.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
